affected-paths/core/src/test/kotlin/com/squareup/affected/paths/core/git/AffectedPathsTest.kt

```diff
@@ -0,0 +1,312 @@
+package com.squareup.affected.paths.core.git
+
+import com.squareup.affected.paths.core.CoreAnalyzer
+import com.squareup.affected.paths.core.CoreOptions
+import kotlinx.coroutines.test.runTest
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+import kotlin.test.Test
+import kotlin.test.assertContentEquals
+import kotlin.time.Duration.Companion.minutes
+
+class AffectedPathsTest {
+
+    @TempDir
+    lateinit var root: File
+
+    @Test
+    fun `Basic affected paths test`() = runTest(timeout = 3.minutes) {
```

Comment:
this test name isn't great. What is it testing? What does "basic" mean?

Reply:
You're right, I need to be more descriptive on this.